docs/guides/connecting_to_your_data/how_to_create_a_batch_of_data_from_an_in_memory_spark_or_pandas_dataframe.md

```diff
@@ -39,12 +39,12 @@ What used to be called a “Batch” in the old API was replaced with <Technical
 
      First, import these necessary packages and modules.
 
-     ```python file=../../../tests/integration/docusaurus/connecting_to_your_data/how_to_create_a_batch_of_data_from_an_in_memory_spark_dataframe.py#L2-L11
+     ```python name="tests/integration/docusaurus/connecting_to_your_data/how_to_create_a_batch_of_data_from_an_in_memory_spark_dataframe.py imports"
      ```
 
      Load an on-disk <TechnicalTag tag="data_context" text="Data Context" /> (ie. from a `great_expectations.yml` configuration) via the `get_context()` command:
 
-     ```python file=../../../tests/integration/docusaurus/connecting_to_your_data/how_to_create_a_batch_of_data_from_an_in_memory_spark_dataframe.py#L14
+     ```python # <snippet name="tests/integration/docusaurus/connecting_to_your_data/how_to_create_a_batch_of_data_from_an_in_memory_spark_dataframe.py get_context"
```

Comment:
Not showing up for some reason: 
![image](https://user-images.githubusercontent.com/9903066/220690069-d4b06dff-c055-4a0b-a2d3-bbe7d8b1abeb.png)